geometries/plan2pyr-AB.smi

```diff
@@ -0,0 +1,24 @@
+CNC N1
```

Comment:
Should we call this folder molecules instead of geometries? `smi` files aren't geometries...

Reply:
created an issue for after this code gets updated